README.md

```diff
@@ -5,3 +5,16 @@ npm install mx-react-components
 
 ## Usage
 [See Documentation](http://moneydesktop.github.io/mx-react-components/)
+
+## Development
+
+To view your documentation changes before committing you can run the following commands:
+
+```
+npm install
+npm run build
```

Comment:
Here's an idea -- if we install and save `webpack-dev-server` then use the command `webpack-dev-server --inline --hot` in place of the jeckyll stuff then we actually don't need jeckyll installed locally and get hot module replacement.